psa-crypto-sys/src/types.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+#![allow(non_camel_case_types)]
+
```

Comment:
I think some of those types are not defined in the spec but just in Mbed Crypto. I think one should be able to find any of those types in the PSA Crypto document. For example `psa_app_key_id_t` is implementation defined I think.

Reply:
I think we'll need a feature to cover the SE-related functionality too, but I won't add that now